src/main/resources/db/changelog/ddl/0008_EIP1-2833_alter_delivery.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0008_EIP1-2833_alter_delivery - add address_format column" context="ddl">
+
+        <addColumn tableName="delivery">
+            <column name="address_format" type="varchar(20)" value="UK">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+
+        <rollback>
+            <dropColumn
+                tableName="delivery"
+                columnName="address_format"
+            />
+        </rollback>
+    </changeSet>
+
+    <changeSet author="nathan.russell@valtech.com" id="0008_EIP1-2833_alter_delivery - rename delivery_method column" context="ddl">
+        <update tableName="delivery">
+            <column name="delivery_method" value="REGISTERED" />
+        </update>
+
+        <renameColumn
+            tableName="delivery"
+            oldColumnName="delivery_method"
+            newColumnName="address_type"
+            columnDataType="varchar(20)"
```

Comment:
Similar comment as above, @NathanRussellValtech. 
We have set this column size to 50 in VCA, so it might be good to set it to 50 here as well for consistency.

Reply:
Yeah, OK, good shout. Will do 👍

Reply:
On a further note, I think it will also be better if we have the same column name as VCA here (`delivery_address_type`). We have delivery_address_type in the VCA table and I saw the sqs message in this PR also has the name `DeliveryAddressType`, so it will be easier to understand the purpose of the column and be more consistent if we have the same naming here as well.

Reply:
I've done as suggested, but I have left the rollback at `varchar(20)` because thats what it is before this change (ie. rolling back to the same state that it is beforehand)